src/languages/en.js

```diff
@@ -1,4 +1,233 @@
+/* eslint-disable max-len */
 export default {
+    common: {
+        cancel: 'Cancel',
+        upload: 'Upload',
+        yes: 'Yes',
+        no: 'No',
+        attachment: 'Attachment',
+        to: 'To',
+        optional: 'Optional',
+        split: 'Split',
+        new: 'NEW',
+        search: 'Search',
+        next: 'Next',
+        add: 'Add',
+        resend: 'Resend',
+        save: 'Save',
+        password: 'Password',
+        profile: 'Profile',
+        payments: 'Payments',
+        preferences: 'Preferences',
+        view: 'View',
+        not: 'Not',
+        signIn: 'Sign In',
+        continue: 'Continue',
+        phoneNumber: 'Phone Number',
+        email: 'Email',
+        and: 'and',
+    },
+    attachmentPicker: {
+        cameraPermissionRequired: 'Camera Permission Required',
+        expensifyDoesntHaveAccessToCamera: 'Expensify.cash does not have access to your camera, please enable the permission and try again.',
+        attachmentError: 'Attachment Error',
+        errorWhileSelectingAttachment: 'An error occurred while selecting an attachment, please try again',
+        errorDuringAttachmentSelection: 'Error during attachment selection',
```

Comment:
aren't `errorWhileSelectingAttachment` and `errorDuringAttachmentSelection` pretty much the same thing said differently